components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java

```diff
@@ -1651,7 +1653,8 @@ public static void removeAspect(String path, String aspect, Registry registry)
                         || propKey.startsWith("registry.custom_lifecycle.checklist.")
                         || propKey.startsWith("registry.LC.name")
                         || propKey.startsWith("registry.lifecycle.")
-                        || propKey.startsWith("registry.Aspects")) && propKey.contains(aspect)) {
+                        || propKey.startsWith("registry.Aspects"))
+                        && propKey.contains(GovernanceConstants.DOT + aspect + GovernanceConstants.DOT)) {
```

Comment:
Thanks for pointing it out. Fixed with the following commit - https://github.com/wso2/carbon-governance/commit/a9913cb98c71985adbcc934464c3a519e1ab3b92